src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/RECAPTCHAManager.java

```diff
@@ -0,0 +1,81 @@
+package uk.ac.cam.cl.dtg.segue.api.managers;
+
+import com.google.inject.Inject;
+import org.apache.commons.lang3.Validate;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.ac.cam.cl.dtg.util.PropertiesLoader;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static uk.ac.cam.cl.dtg.segue.api.Constants.GOOGLE_RECAPTCHA_SECRET;
+
+public class RECAPTCHAManager {
+    private final PropertiesLoader properties;
+    private static final Logger log = LoggerFactory.getLogger(RECAPTCHAManager.class);
+
+    @Inject
+    public RECAPTCHAManager(final PropertiesLoader properties) {
+        Validate.notNull(properties.getProperty(GOOGLE_RECAPTCHA_SECRET));
+        this.properties = properties;
+    }
+
+    public String isCaptchaValid(final String response) {
```

Comment:
are we relying purely on string content here to denote whether a captcha has been validated?

Reply:
Yes, we check first to see if the string is not empty and then we verify the string. With the external api call to google reCAPTCHA.